frame/bonded-finance/src/weights.rs

```diff
@@ -0,0 +1,24 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(clippy::unnecessary_cast)]
+
+use frame_support::weights::Weight;
+
+pub trait WeightInfo {
+	fn offer() -> Weight;
+	fn bond() -> Weight;
+	fn cancel() -> Weight;
+}
+
+impl WeightInfo for () {
+	fn offer() -> Weight {
+    0 as Weight
```

Comment:
i usually put here something. so that any test will consider that - so our tests become more real:)

Reply:
:+1: